app/assets/stylesheets/styles/components/crops/components/tags-input.scss

```diff
@@ -0,0 +1,399 @@
+                  line-height: 21px;
+                  font-size: 12px;
+                  border-radius: 0px;
+                }
+              }
+            }
+          }
+
+          .remove-button {
+
+            .input-group-sm {
+
+              .keywords {
+
+                .tags {
+
+                  .tag-item {
+
+                    .remove-item {
+
+                      font-size: 16px;
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        .input {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
+
+                .input {
+                  height: 22px;
+                  font-size: 12px;
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+.keywords {
+
+  .tags {
+
+    .tag-item {
+
+      &.selected {
+        background: -webkit-linear-gradient(top, #febbbb 0%, #fe9090 45%, #ff5c5c 100%);
+        background: linear-gradient(to bottom, #febbbb 0%, #fe9090 45%, #ff5c5c 100%);
+      }
+
+      .remove-button {
+        margin: 0 0 0 5px;
+        padding: 0;
+        border: none;
+        background: none;
+        cursor: pointer;
+        vertical-align: middle;
+        font: bold 16px Arial, sans-serif;
+        color: #585858;
+
+        &:active {
+          color: #ff0000;
+        }
+      }
+    }
+
+    .tag-list {
+      padding: 0.07em;
+    }
+
+    .input {
+      margin: 1px;
+      font: inherit;
+      font-size: 0.88rem;
+      border-radius: 0px;
+    }
+  }
+}
+
+.keywords[disabled] {
+
+  .tags {
+    background-color: #eeeeee;
+
+    .tag-item {
+      background: #337ab7;
+      opacity: 0.65;
+    }
+
+    .input {
+      background-color: #eeeeee;
+    }
+  }
+}
+
+.input-group {
+
+  .keywords {
+    padding: 0;
+    display: table-cell;
+  }
+}
+
+.input-group .keywords:not(:first-child) .tags {
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+}
+
+.input-group .keywords:not(:last-child) .tags {
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.input-group-lg .keywords:first-child .tags {
+  border-top-left-radius: 0px;
+  border-bottom-left-radius: 0px;
+}
+
+.input-group-lg .keywords:last-child .tags {
+  border-top-right-radius: 0px;
+  border-bottom-right-radius: 0px;
+}
+
+.input-group-sm .keywords:first-child .tags {
+  border-top-left-radius: 0px;
+  border-bottom-left-radius: 0px;
+}
+.input-group-sm .keywords:last-child .tags {
+  border-top-right-radius: 0px;
+  border-bottom-right-radius: 0px;
+}
+
+.has-feedback .keywords .tags {
+  padding-right: 30px;
+}
+
+.has-success .keywords .tags {
+  border-color: #cccccc;
+
+  &.focused {
+    border-color: #cccccc;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #67b168;
+    -moz-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #67b168;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 6px #67b168;
```

Comment:
You can also look at https://robots.thoughtbot.com/controlling-color-with-sass-color-functions#adding-alpha-transparency